Blazor.Test/Program.cs

```diff
@@ -0,0 +1,34 @@
+using Blazor.Test;
+using Microsoft.AspNetCore.Components.Web;
+using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
+using Microsoft.JSInterop;
+using System.Globalization;
+
+var builder = WebAssemblyHostBuilder.CreateDefault(args);
+builder.RootComponents.Add<App>("#app");
+builder.RootComponents.Add<HeadOutlet>("head::after");
+
+builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+
+var host = builder.Build();
+
+CultureInfo culture;
+var js = host.Services.GetRequiredService<IJSRuntime>();
+var result = await js.InvokeAsync<string>("blazorCulture.get");
+
+if (result != null)
+{
+    culture = new CultureInfo(result);
+}
+else
+{
+    culture = new CultureInfo("en-CA");
+    await js.InvokeVoidAsync("blazorCulture.set", "en-CA");
+}
+
+CultureInfo.DefaultThreadCurrentCulture = culture;
+CultureInfo.DefaultThreadCurrentUICulture = culture;
+
+await js.InvokeVoidAsync("installCDTS", CultureInfo.CurrentCulture.Parent.IetfLanguageTag);
```

Comment:
Just a thought: Could this be an async .Net function where we do this for them (I guess they'd have to pass in `js`).   I'm just thinking that 1- they wouldn't need to know the name of the Javascript function, 2- they wouldn't be affect if we rename Javascript stuff and 3- if we want to add some other init-time stuff, that function will be a place where we can do that.